cli.js

```diff
@@ -64,8 +65,10 @@ function render() {
 function setState(s) {
 	state = s;
 
-	if (s && s.length > 0) {
+	if (s && s.length > 0 && cli.flags.bytes !== true) {
```

Comment:
Instead of doing the MBps vs Mbps check every, just do it once and use a variable everywhere.